docs/flow/modin/core/execution/python/implementations/pandas_on_python/index.rst

```diff
@@ -1,5 +1,32 @@
+PandasOnPython Execution
+========================
+
+Queries that perform data transformation, data ingress or data egress using the `pandas on Python` execution
+pass through the Modin components detailed below.
+
+To enable `pandas on Python` execution, please refer to the usage section in :doc:`pandas on Python </UsingPandasonPython/index>`.
```

Comment:
I feel we should duplicate some info here about `PandasOnPython` being sequential engine with main purpose being debugging.

Also this link wasn't generated for some reason.

Reply:
I agree, info about sequential execution/debugging is added.

Link isn't generated because the doc </UsingPandasonPython/index> doesn't exist. This is future doc. The same you can see for Ray executiuon: https://github.com/modin-project/modin/blame/master/docs/flow/modin/core/execution/ray/implementations/pandas_on_ray/index.rst#L7 (link on </UsingPandasonRay/index> isn't generated also).